Sources/WalletConnectSign/Sign/Sign.swift

```diff
@@ -157,8 +157,8 @@ extension Sign {
     /// - Parameters:
     ///   - proposal: Session Proposal received from peer client in a WalletConnect delegate.
     ///   - reason: Reason why the session proposal was rejected. Conforms to CAIP25.
-    public func reject(proposal: Session.Proposal, reason: RejectionReason) {
-        client.reject(proposal: proposal, reason: reason)
+    public func reject(proposal: Session.Proposal, reason: RejectionReason) throws {
```

Comment:
just a note: we are currently approving by `proposalId`(aka: proposer.publicKey) so maybe we should also rejecting by `proposalId` instead of requiring the whole Proposal object.